vm/src/pyobject.rs

```diff
@@ -364,7 +373,15 @@ impl PyContext {
         self.types.object_type.clone()
     }
 
+    #[inline]
     pub fn new_int<T: Into<BigInt>>(&self, i: T) -> PyObjectRef {
+        let i = i.into();
```

Comment:
I was thinking that instead of immediately converting it, `T` could be `Into<BigInt> + PartialOrd<i32>`, to save on the overhead of allocation for a BigInt.

Reply:
Because `PartialOrd<i32>` is not implemented for unsigned ints, this way will be tricky

Reply:
Maybe `T` could be `ToPrimitive`?

Reply:
This is great idea. I was struggling with TryInto and TryFrom but ToPrimitive is a lot easier to do this. Thanks!